.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs) GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new topics.
+
+Magento's tech writer team and [Maintainers](https://devdocs.magento.com/contributor-guide/devdocs-maintainers.html) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards and thanks
+
+DevDocs works with the Community Engineering teams and projects.
+As you contribute PRs, you gain [Contribution Points](https://github.com/magento/magento2/wiki/Contribution-Rewards).
+
+If you write and contribute a full topic, we will put your name (or your company's name) at the top of the page right under the title, and link it to your blog or website!
+We track and post your picture and a link to your GitHub account on the [Top recent contributors](https://devdocs.magento.com/guides/v2.2/contributor-guide/quarterly-contributors.html) page.
+
+## Get started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* Make sure you sign the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca).
+* Check the [guidelines](#contribution-guidelines).
+* [Fork and clone](https://help.github.com/articles/fork-a-repo/) the [DevDocs repository](GitHub.com/magento/devdocs). [Sync](https://help.github.com/articles/syncing-a-fork/) as needed.
+* Write content and commit.  Need help, see our [templates](#templates).
+* Create a [pull request](https://help.github.com/articles/creating-a-pull-request/). Fill out as much info as possible and link any GitHub issues.
+
+**Note:** If you have not signed the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca), the PR will provide a link to complete. All contributors are required to submit the form and agree to the terms.
```

Comment:
Should the Note and Tip be in a code formatted style like the ones we typically use on DevDocs?